index.html

```diff
@@ -392,53 +439,53 @@ <h4>Interchain Network</h4>
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <h5 class="modal-title">CLC Token Faucet</h5>
+                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                            <span aria-hidden="true">&times;</span>
+                        </button>
+                    </div>
+                    <div class="modal-body" style="text-align: center;">
+                        <div class="container-fluid">
+                            <div class="row">
+                                <div class="col">
+                                    <p>Send ETH using your favourite wallet to the following contract address to automatically receive CLC tokens at the displayed rate.</p>
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col-md-6">
+                                    <img style="margin: 0px;" height="250" width="250" src="./dist/img/faucet-qr.png" alt="faucet-address-qr-code"/>
+                                    <p><small>0xB36502c671C1157b5989d89E20bB96be98465116</small></p>
+                                </div>
+                                <div class="col-md-6 align-self-center">
+                                    <div class="row">
+                                        <div class="col">
+                                            <h4>Exchange Rate:</h4>
+                                            <p>1 ETH : 15000 CLC</p>
+                                        </div>
+                                    </div>
+                                    <div class="row">
+                                        <div class="col">
+                                            <h4>Exchange Limit:</h4>
+                                            <p>2 ETH</p>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col">
+                                    <div class="alert alert-warning" role="alert">
+                                        <p><b>Disclaimer: </b>Do not send ETH to faucet directly from an exchange or contract or you may not receive your CLCs.</p>
+                                        <small>Presently the faucet is restricted from users in the following regions: Mainland China, America, Bosnia and Herzegovina, North Korea, Ethiopia, Iran, Iraq, Sri Lanka, Syria, Trinidad and Tobago, Tunishia, Vanuatu, Yemen. </small>
+                                    </div>
+                                </div>
+                            </div>
                         </div>
-                      </div>
                     </div>
-                  </div>
```

Comment:
You are still deleting contents that are irrelevant for the change. And its still really hard to see what is changed given all the format updates.